packages/codemirror-extension/src/services.tsx

```diff
@@ -200,15 +200,15 @@ export const extensionPlugin: JupyterFrontEndPlugin<IEditorExtensionRegistry> =
   };
 
 /**
- * CodeMirror real-time collaboration binding provider.
+ * CodeMirror binding provider.
```

Comment:
Could we keep a reference to what it is bound to CodeMirror?

```suggestion
 * CodeMirror shared model binding provider.
```